keyboards/nullbitsco/nibble/nibble_encoder.h

```diff
@@ -0,0 +1,140 @@
+#include QMK_KEYBOARD_H
+#include "tmk_core/common/eeprom.h"
```

Comment:
Unsure if this is required, but......
```suggestion
#include "eeprom.h"
```